packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -309,16 +273,9 @@ test.describe('PieCookieBanner - Component tests', () => {
     });
 
     test.describe('`hasPrimaryActionsOnly` prop', () => {
-        test('should set both buttons to primary when true', async ({ mount }) => {
+        test('should set both buttons to primary when true', async () => {
             // Arrange & Act
-            await mount(
-                PieCookieBanner,
-                {
-                    props: {
-                        hasPrimaryActionsOnly: true,
-                    },
-                },
-            );
+            await pieCookieBannerComponent.load({ hasPrimaryActionsOnly: true });
 
             // Act
             const acceptAllButtonVariant = await pieCookieBannerComponent.getAcceptAllVariant('variant');
```

Comment:
Not part of this PR, but the naming of this looks a bit off `getAcceptAllVariant('variant')` sounds like it has some redundancy in there.